### PR TITLE
Add Feature availability and Command-line upgrade info to release notes

### PR DESCRIPTION
**Pull request by shrielenee**
## This PR is a:

- [ ] New topic
- [X] Content update
- [ ] Content fix or rewrite
- [ ] Bug fix or improvement

## Summary

When this pull request is merged, it will:
* Move the pre-release/install info to top section
* Remove note about impending beta code because it is currently available
* Revise Prerequisites link in Command-line upgrade doc, it was broken
* Add a note to the top of the 2.3 install guide about the Command-line upgrade instructions

## Additional information

List all affected URLs 

- https://devdocs.magento.com/guides/v2.3/comp-mgr/cli/cli-upgrade.html#pre-upgrade-checklist
- https://devdocs.magento.com/guides/v2.3/release-notes/2.3.0-quick-start.html
- https://devdocs.magento.com/guides/v2.3/release-notes/2.3.0-install.html

### CODE REVIEW

File: guides/v2.3/release-notes/2.3.0-quick-start.md [hunk 1]
```diff
@@ -16,19 +19,14 @@ This guide covers:
 
 * **Submit comments on Magento 2.3.0 Beta code** on [GitHub](https://github.com/magento/magento2/issues)
 
-* **Readiness status of individual code components**. Magento 2.3.0 pre-release code is a work in progress. Consequently, parts of the code base are more mature than others. You can see an overview of component status at [Component Status]({{page.baseurl}}/release-notes/component-status.html). Note that we will update this information weekly during the Release evaluation period.
-
-
-{:.bs-callout .bs-callout-note}
-PageBuilder Beta code will be available in 2018 Q4. Registered participants will be able to install PageBuilder Beta on Magento 2.3.0 Commerce code.  Watch this space for more information about participating in the PageBuilder Beta program plus installation instructions.
```
[the remark this answers is not in the document]
Comment (shrielenee) [reply guides/v2.3/release-notes/2.3.0-quick-start.md]: Beta is available now, so the removed note is incorrect as its currently available.

The Component status page shows Green, which only means the module has no significant or outstanding issues, not that it is or isn't available. So I think we can leave that, no?

I added a link to the availability page within that note.

**Comment by shrielenee** [comment]
I agree about the feature being popular, but the note says that Beta "will be available in 2018 Q4". Isn't it already available? If so, should I just reword the note and leave it? Please advise @lorikrell.

**Comment by lorikrell** [comment]
Not a problem, Shrie! You have a deeper knowledge of Pagebuilder, just wanted to ask. Feel free to remove. 

Go for it!

OUTSIDE OF THIS PR: As to the component page, it will need a review. There's missing modules, and one of the listed is deprecated (may be worth adding a note to that one or something). Or a long term discussion around deprecated, but not removed yet, modules?

**Comment by shrielenee** [comment]
Jira issue MAGEDOC-3226 created regarding review of Component Status page.

**Pull request merged**